README.md

```diff
@@ -55,12 +61,79 @@ Most attempts at addressing this complexity restricts a schema to only a certain
 
-## Building MEI Schemas
-
-Building MEI requires the [TEI Stylesheets](https://github.com/TEIC/Stylesheets/). You should clone their git repository, or download them in a packaged zip file.
+# Building MEI
 
-To build the CMN customization, for example, you can use the `teitorelaxng` command in the TEI Stylesheets. Let's assume that your stylesheets are in /opt/TEI, and your MEI source is in /opt/MEI:
+The MEI Source is not a schema in itself; rather, it can be used to build customized schemas, such as mei-CMN, mei-Mensural, mei-all, etc. (also see [Customizing MEI](https://github.com/music-encoding/music-encoding#customizing-mei).
+
+Nevertheless it is possible to build any customization locally in your working copy of this repository. In order to do so follow the steps below:
+
+1. Create a recursive clone of this repository
+
+   * If you do not have a clone on your local machine yet run the following from your command line:
+
+     ```bash
+     git clone https://github.com/music-encoding.git --recursive
+     ```
+     
+   * If you already have a clone on your system you still might have to initialize the submodules by running the following commands form the command line:
+
+     Switch to your clone's directory:
+     ```bash
+     cd [YOUR-CLONE-LOCATION]
+     ```
+     Initialize the submodules:
+     ```bash
+     git submodules init
+     ```
+     Update the submodules:
+     ```bash
+     git submodules update
+     ```
+2. Check if your system meets the build requisites
+
+   * Is Java 8 or above available on your machine?
+   
+     ```bash
+     java -version
+     ```
+   
+   * Is Apache Ant installed?
+
+     ```bash
+     ant -version
+     ```
+   
+2. Initialize the build process
+
+   * Switch to your clone's directory:
+   ```bash
+   cd [YOUR-CLONE-LOCATION]
+   ```
+   
+   * Call the Apache Ant init task:
+   ```bash
+   ant init
+   ```
+
+4. Run the build process
+
+   * Build guidelines HTML:
+   ```bash
+   ant -lib lib/saxon/saxon9he.jar build-guidelines-html
+   ```
+
+   * Build a specific customization's RNG schema:
+   ```bash
+   ant -lib lib/saxon/saxon9he.jar -Dcustomization.path=[PATH/TO/YOUR/CUSTOMIZATION] build-rng
+   ```
+
+   * Build everything (all customizations shipped with tis repository, compiled ODDs for each customization, guidelines HTML):
+   ```bash
+   ant -lib lib/saxon/saxon9he.jar
+   ```
+
+# Additional Resources
+
+Samples of MEI-encoded files are available in the [sample-encodings](https://github.com/music-encoding/sample-encodings) repository, and tools for working with and converting to and from other formats and MEI are available in the [encoding-tools](https://github.com/music-encoding/encoding-tools) project.
```

Comment:
> ..., and tools for working with and converting to and from other formats and MEI ...

That's not very easy to read. Maybe:

..., and a number of helpful tools for working with MEI documents (mainly XSLT Stylesheets for transformation of encodings or for conversion to and from other formats) are available...

Reply:
Additionally, this information is still doubled in L28?

Reply:
yeah just copied it, probably it would be best to move all the Additional Resource stuff down and link from here to the corresponding section

Reply:
guess that's fixed